build.gradle

```diff
@@ -41,16 +41,36 @@ allprojects {
         mavenCentral() { metadataSources { mavenPom(); ignoreGradleMetadataRedirection() } }
     }
 
-    sourceCompatibility = 11
-    targetCompatibility = 11
-
     group 'com.palantir.common'
     version System.env.CIRCLE_TAG ?: gitVersion()
+
+    tasks.withType(Test).configureEach {
```

Comment:
Would prefer to use defaults here, otherwise there’s some risk that we stop running tests when things change

Reply:
removed these, I had copied this over to locally compare test count